### PR TITLE
complete page layout

### PR DESCRIPTION
**Pull request by markpsyntax**
#What this does
Initial page layout, normally I would break this down into smaller steps and do more frequent pr's but for the sake of time I did 1 bulk pr. Also a bit of the code doesn't follow the best DRY practices, but again for the sake of time.

#Pictures of change for FE
Screenshot 2023-06-28 at 2 19 26 PM

### CODE REVIEW

**Comment by markpsyntax** [comment]
Final update to code, this runs the calculation and logs the proper response in the network tab. We can see by this image below that the data is being properly logged
Screenshot 2023-06-28 at 7 10 16 PM

I have updated the response page to show data like so:
Screenshot 2023-06-28 at 7 09 45 PM

@raytiley let me know if you'd like me to update any of the code or if any of the data isn't in the correct format.

**Comment by markpsyntax** [comment]
Hey @raytiley I was doing one last look over the project this morning, did you want it to be mobile responsive? Also, did you want it hosted or is the code by itself fine?

**Comment by raytiley** [comment]
Hey Mark,

We do our best to make things work on mobile devices where practical. Since this is a marketing tool, lots of users probably clicking through on their phone it is something we would for sure consider. Realizing that I only gave you a desktop mockup, do your best :)

**Comment by raytiley** [comment]
> Also, did you want it hosted or is the code by itself fine?

Code by itself if fine. I'll pull it down and run it locally tomorrow. Thanks!

**Comment by markpsyntax** [comment]
Some updates for mobile users

Screenshot 2023-06-29 at 10 12 04 AM
Screenshot 2023-06-29 at 10 12 12 AM
Screenshot 2023-06-29 at 10 12 29 AM

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # Cablecast Web Developer Take Home Assignment
 
-Thanks for taking the time to apply with us. We really appreciate it. The goal of this project is to guage how you work the way we work, remotely, largely indepenently, colobarating as needed. If you run into any issues at all with this project or have any questions please reach out to @raytiley.
+Thanks for taking the time to apply with us. We really appreciate it. The goal of this project is to gauge how you work the way we work, remotely, largely independently, collaborating as needed. If you run into any issues at all with this project or have any questions please reach out to @raytiley.
```
Comment (raytiley) [inline README.md:3]: Thanks for fixing my spelling :)

**Pull request closed without merging**